README.md

```diff
@@ -1,2 +1,228 @@
-# satellite6-upgrade
-Satellite 6 upgrade related tools
+# Satellite6-upgrade
+
+Satellite6 upgrade is a set of tools to be used for following reasons:
+1. Creating and deleting satellite/capsule instances using RHEVM templates/Openstack images
+2. Upgrade Satellite6, Capsule and clients (RHEVM instances / Users own setup)
```

Comment:
`2. Upgrade Satellite6, capsule instances spawn from RHEVM templates and clients generated through docker`
`3. In case user has already an existing satellite/capsule setup and wants to perform upgrade then Satellite6 upgrade has the capability to perform upgrade on user provided setups`